editor/plugins/polygon_2d_editor_plugin.cpp

```diff
@@ -1052,6 +1056,21 @@ void Polygon2DEditor::_uv_draw() {
 		return;
 	}
 
+	// Force point editing mode if there's no polygon yet.
+	if (node->get_polygon().size() == 0) {
+		if (!uv_edit_mode[1]->is_pressed()) {
+			uv_edit_mode[1]->set_pressed(true);
+			_uv_edit_mode_select(1);
```

Comment:
Not sure why is it needed. `pressed` signal should handle it, but it doesn't work here.

Reply:
I thought so too. Go figure 🤷‍♂️

Reply:
Ok I figured it.
`set_pressed()` only emits `toggled` signal.